assets/js/modules/analytics-4/datastore/audience-settings.js

```diff
@@ -257,6 +271,10 @@ const baseSelectors = {
 	getAudienceSettings( state ) {
 		return state.audienceSettings;
 	},
+
+	getAvailableAudiencesLastSyncedAt( state ) {
+		return state.audienceSettings?.availableAudiencesLastSyncedAt;
+	},
```

Comment:
Hi @ankitrox, I've just spotted another thing that we need to fix here, that should have been addressed in a previous issue.

All these individual audience setting selectors need to call `getAudienceSettings()` in order to invoke its resolver. So, this one should be updated as follows:

```suggestion
	/**
	 * Gets the last time the available audiences were synced.
	 *
	 * @since n.e.x.t
	 *
	 * @param {Object} state Data store's state.
	 * @return {(number|undefined)} Last time the available audiences were synced, or `undefined` if not loaded.
	 */
	getAvailableAudiencesLastSyncedAt: createRegistrySelector(
		( select ) => () => {
			const audienceSettings =
				select( MODULES_ANALYTICS_4 ).getAudienceSettings() || {};

			return audienceSettings.availableAudiencesLastSyncedAt;
		}
	),
```

Please can you apply this to the other selectors too? Also, don't forget the JSDoc :)